scripts/operator/raise_pr_to_community_operators_from_our_fork.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/python3
+
+import os
+from git import Repo
+from github import Github
+import shutil
+from get_last_published_operator_version import getLastOperatorVersion
+import pr_resources
+
+
+def get_latest_operator_branch():
+    repo = Repo.clone_from(
+        'https://github.com/snyk/community-operators.git', './community-operators-repo')
+
+    operator_branches = []
+
+    for refs in repo.refs:
+        if 'snyk-operator' in refs.name:
+            operator_branches.append(refs.name)
+
+    return operator_branches[-1].split('origin/')[-1]
```

Comment:
are we sure this logic is correct? how is it ordered?

Reply:
yes, I've tested it adding a new branch, it looks like it's always last